docs/mgiration.md

```diff
@@ -18,47 +18,47 @@ This tool runs on RKE1 nodes before running RKE2.
 
 ### Usage
 
-1- To run the migration you need to take a snapshot first on rke1 nodes:
-
-```
-rke etcd snapshot-save --s3 --name rke1snapshot --access-key <access-key> --secret-key <secret-key> --region <region> --folder <folder> --bucket-name <bucket name>
-```
-
+1. To run the migration you need to take a snapshot first on rke1 nodes:
+	
+	    ```
+	    rke etcd snapshot-save --s3 --name rke1snapshot --access-key <access-key> --secret-key <secret-key> --region <region> --folder <folder> --bucket-name <bucket name>
+	    ```
+	
```

Comment:
these need to be indented exactly 4 spaces for the markdown to format properly. It looks like you're using tabs?

Same for all the sections below.